cmd/ttn-lw-stack/commands/start.go

```diff
@@ -187,6 +191,12 @@ var (
 				rootRedirect = web.Redirect("/", http.StatusFound, config.Console.UI.CanonicalURL)
 			}
 
+			if start.GCS {
+				logger.Info("Setting up GCS")
```

Comment:
Bit late now, but this should have been written out: Gateway Configuration Server

Reply:
I'll sneak this in the next PR.